charts/opentelemetry-operator/README.md

```diff
@@ -49,10 +49,16 @@ $ helm install --namespace opentelemetry-operator-system \
 If you wish for helm to create an automatically generated self-signed certificate, make sure to set the appropriate values when installing the chart:
 
 ```console
-$ helm install  --set admissionWebhooks.certManager.enabled=false --set admissionWebhooks.certManager.autoGenerateCert=true \
+$ helm install --set admissionWebhooks.certManager.enabled=false --set admissionWebhooks.certManager.autoGenerateCert=true \
   opentelemetry-operator open-telemetry/opentelemetry-operator
 ```
 
+Additional featureGates can also be enabled through the Helm installation. For enabling the [rewrite target allocator](https://github.com/open-telemetry/opentelemetry-operator#target-allocator-config-rewriting) feature, add a `--set` flag as follows.
```

Comment:
Lets move this into a new Operator Feature Gates section of the readme nested under the `## Configuration` section.

Reply:
Sure thing